ui/src/app/views/admin-review-workspace/component.html

```diff
@@ -34,4 +34,6 @@ <h3 class="modal-title">Error:</h3>
     <span class="loading-text">Loading workspaces for review...</span>
   </div>
 </ng-template>
-<app-bug-report></app-bug-report>
\ No newline at end of file
+<app-bug-report *ngIf="bugReportOpen"
```

Comment:
I don't know if we actually have this automatic bug report thing anywhere else. This may be able to just be removed.